### ISSUE TITLE
saving a single image for each timestep.

### ISSUE THREAD

**Issue by oceandiff** [OpenDrift/opendrift#765]
Hello,
is it possible to save a single image for each timestep in a specific directory?
thankyou

**Comment by knutfrode** [comment]
No, this is not possible. But you can make an animation (mp4 or gif) and convert to image for each frame.

**Comment by oceandiff** [comment]
Ok, thanks.
BTW, is possible to save the data in a NetCDF and post-process it, right?

**Comment by knutfrode** [comment]
Yes, just add `outfile=file.nc` to `run` method.
